templates/object-cache.php

```
@@ -64,6 +71,21 @@ public function __construct() {
+		 *
+		 * Called by wp_cache_add_salt() (WP 6.9+) to invalidate all cached data
+		 * by changing the key space.
+		 *
+		 * @since 2.1.0
+		 *
+		 * @param string $salt The salt string to add.
+		 * @return void
+		 */
+		public function add_salt( $salt ) {
+			$this->salt = $salt;
+		}
```

PHPMD (2.15.0)

[error] The method add_salt is not named in camelCase. (undefined)

(CamelCaseMethodName)